fuse/src/main/java/com/github/kittinunf/fuse/core/Cache.kt

```diff
@@ -30,16 +30,20 @@ object CacheBuilder {
     }
 }
 
-fun <T : Any> Config<T>.build() = Cache(this)
+fun <T : Any> Config<T>.build(): Cache<T> = CacheImpl(this)
 
-class Cache<T : Any> internal constructor(private val config: Config<T>) : Fuse.Cacheable,
-    Fuse.Cacheable.Put<T>, Fuse.Cacheable.Get<T>, Fuse.DataConvertible<T> by config.convertible {
+enum class Source {
```

Comment:
Moved out of the concrete class, to the top level. This makes reusing of this enum more natural in alternative concrete implementations of `Cache`.